python/adelphi/adelphi/anonymize.py

```diff
@@ -1,12 +1,10 @@
+import re
 # Functions and constants related to the anonymization process
-
 from adelphi.store import get_standard_columns_from_table_metadata
 
 # default prefixes for the anonymized names
 KEYSPACE_PREFIX = "ks"
 TABLE_PREFIX = "tbl"
-PARTITION_KEY_PREFIX = "pk"
-CLUSTERING_KEY_PREFIX = "ck"
```

Comment:
Splitting pk/ck from regular columns added some unneeded complexity (some names might be clustering keys in one schema but not in another). The original requirement was to anonymize all columns to `col_`, so I converged to that.